src/components/LocationSearch/location-search.module.css

```diff
@@ -0,0 +1,69 @@
+.wrapper {
+  box-shadow: 0 2px 4px 2px rgba(0, 0, 0, 0.1);
+}
+
+.label {
+  /* Visually Hidden */
```

Comment:
we have a `helpers.module.css` class `.visuallyHidden` — not that it really matters, since we'll be throwing all that away.

Reply:
I've left this for now as we'll be rewriting with Emotion and have a nice component for this :)

Reply:
exactly :)